kernel/Dockerfile

```diff
@@ -18,7 +19,7 @@ RUN if [ $DEBUG -ne "0" ]; then \
     fi
 
 # Apply local patches
-COPY patches-4.9 /patches
+COPY patches-${KERNEL_SERIES} /patches
```

Comment:
there doesn't seem to be an easy solution to this. There were proposals (e.g. https://github.com/moby/moby/issues/13045) and patches (e.g. https://github.com/moby/moby/pull/13813) to ignore errors but they weren't accepted.

This is also not a regression to what we had before, and I would suggest to ignore it for now and address, if we hit it. A possible workaround would be always create the directory and then later check that `*.patches` is not empty...